application/data_access/entity_queries.py

```diff
@@ -90,11 +92,35 @@ def get_entity_search(session: Session, parameters: dict):
     query = _apply_limit_and_pagination_filters(query, params)
 
     entities = query.all()
-    entities = [entity_factory(entity_orm) for entity_orm in entities]
-
+    if exclude_field:
```

Comment:
I don't think you need the code here as you're applying the exclude_entity parameter in the router function already. I think you've made the right decision to edit it there as you're not changing the query being performed and it may not make sense to as the entity factory validates whatever comes back. I think remove this and. keep the code in the router function. to use the parameter there.